modules/nix/default.nix

```diff
@@ -227,6 +227,19 @@ in
                 The hostname of the build machine.
               '';
             };
+            protocol = mkOption {
+              type = types.enum [ null "ssh" "ssh-ng" ];
+              default = "ssh";
```

Comment:
Given that `null` doesn't add any `://` scheme prefix that seems like a better option for backwards compatibility to me. But guess it's fine since that's also that the nixos module uses.